### PR TITLE
chore: add more verbosity to some p2p comm logs

### PR DESCRIPTION
**Pull request by P1sar**
In order to better debug current KeyRefresh problem we I have update some of the logs verbosity.
## Related Issue Or Context

Related: https://github.com/sygmaprotocol/project-backlog/issues/149
Closes: #<issue>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

File: comm/p2p/libp2p.go [hunk 1]
```diff
@@ -191,7 +191,7 @@ func (c Libp2pCommunication) sendMessage(
 
 	err = WriteStream(msg, bufio.NewWriterSize(stream, defaultBufferSize))
 	if err != nil {
-		c.logger.Error().Str("To", string(to)).Err(err).Msg("unable to send message")
+		c.logger.Error().Str("To", to.String()).Err(err).Msg("unable to send message")
```
Comment (mpetrun5) [inline comm/p2p/libp2p.go:194]: I think it should be Pretty(). 
String is formatted weird.

**Review by mpetrun5 — CHANGES_REQUESTED** [review]

File: comm/p2p/libp2p.go (hunk 1 above)
Comment (P1sar) [reply comm/p2p/libp2p.go:194]: But pretty is Deprecated
```
// Pretty returns a base58-encoded string representation of the ID.
// Deprecated: use String() instead.
```

Comment (P1sar) [reply comm/p2p/libp2p.go:194]:
```
func (id ID) Pretty() string {
	return id.String()
}
```

Comment (mpetrun5) [reply comm/p2p/libp2p.go:194]: Oh, I guess it is fine then

**Review by MakMuftic — APPROVED** [review]

**Pull request merged**